test/e2e/cmd/init/init.go

```diff
@@ -374,6 +375,26 @@ func (ki *Initializer) prepareKindConfigFile(kindConfigPath string) error {
 }
 
 func (ki *Initializer) configureAddons() error {
+	err := wait.PollImmediate(10*time.Second, 4*time.Minute, func() (bool, error) {
```

Comment:
how about move this code snippet into `yurt-manager deployment`? so e2e process can go ahead before yurt-manager is ready.

Reply:
Yes, that's better.